contracts/tests/psp34.rs

```diff
@@ -80,35 +82,40 @@ mod psp721 {
             _to: AccountId,
             _id: Id,
             _data: Vec<u8>,
-        ) -> Result<(), PSP721Error> {
+        ) -> Result<(), PSP34Error> {
             Ok(())
         }
     }
 
-    impl PSP721 for PSP721Struct {}
+    impl PSP34 for PSP34Struct {}
 
-    impl PSP721Struct {
+    impl PSP34Struct {
         #[ink(constructor)]
         pub fn new() -> Self {
             Self::default()
         }
     }
 
+    #[ink::test]
+    fn collection_id_fails() {
```

Comment:
https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html#testing-panics

Reply:
thank you